system/security/cryptohelper/digisign.hpp

```diff
@@ -37,6 +37,9 @@ interface IDigitalSignatureManager : extends IInterface //Public/Private key mes
 public:
     virtual bool isDigiSignerConfigured() = 0;
     virtual bool isDigiVerifierConfigured() = 0;
+    virtual void setKeyFileNames(const char * pub, const char * priv) = 0;
```

Comment:
This shouldn't be in the public interface.  It can be a member of the class that is created.

Reply:
agreed